README.md

```diff
@@ -138,14 +140,12 @@ information about how to use a built image with the targets, see
 [Getting started](getting-started.md).
 
 ### Pelux Intel
-Reference instance for the Intel i7 x86 platform. Examples of boards using this
-architecture include:
+Reference instance for the Intel i7 x86 platform. 
 
-* Intel NUC
-* Minnowboard
+* [Intel NUC](https://en.wikipedia.org/wiki/Next_Unit_of_Computing)
```

Comment:
Please keep the Minnowboard here, as it, from a Yocto perspective, uses the i7 platform. Perhaps we can change the wording to just say generic Intel amd64 or something.